app/controllers/api/v1/items_controller.rb

```diff
@@ -2,4 +2,9 @@ class Api::V1::ItemsController < ApplicationController
   def index
     render json: Item.all
   end
+
+  def show
+    item = Item.find(params[:id])
+    json_response(item)
```

Comment:
I suppose there are arguments for both. Since the module is just providing access to a few methods, why would a class be more advantageous? I can see abstracting it out of the ApplicationController as a pro, but a module is really simple for a couple of methods. Thoughts? Unknown complications of a module?

Reply:
Mixins can make it hard to figure out where a method comes from. For example, when I saw `json_response`, my first thought was "Oh! I have never seen this Rails function before". And then only later did I realize that it came from `ApplicationController`. It also means I cannot accidentally declare a "json_response" function in the class otherwise I will clash with your mixin.

Reply:
Ok, this may be a good pairing exercise. I am going to leave it as-is for now.

Reply:
Totally fine to leave it as is. Though admittedly, I don't think there is too much to pair on to refactor this code. Basically, remove `include Whatever`, from your `ApplicationController` and then replace `json_response(...)` with `render json: YourClassName.new.json_response(...)`